policy/config/output/lsstComCamDefault.yaml

```diff
@@ -0,0 +1,47 @@
+output:
+  type: LSST_CCD
+  nproc: 8    # Change this to work on multiple CCDs at once.
```

Comment:
Also,  wouldn't `nproc:9` work better given that there are 9 CCDs in R22 ?

Reply:
Makes sense. I'll change that and eliminate comment below.